### PR TITLE
Add branch 1

### PR DESCRIPTION
**Pull request by AndriySikora**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+<head>
+<link rel="stylesheet" type="text/css" href="style/style.css">
+	<title>Account Form</title>
+</head>
+<body>
+<form id = 'form'>
```
Comment (answer261) [inline index.html]: remove spaces around `=`

File: script/script.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+		alert('Your email is invalid');
+	}
+	clearInputEmail();
+}
+function clearInputEmail(){
+	document.getElementById('email').value = '';
+}
+
+function checkPass(){
+	if (document.getElementById('psw').value.length < '6'){
```
Comment (answer261) [inline script/script.js]: `6` should be a number not a string

File: script/script.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+		return false;
+	}
+	return true;
+}
+
+function passMatch(){
+	var psw = document.getElementById('psw').value;
+	var confirmPsw = document.getElementById('confirmPass').value;
+		if(psw == confirmPsw){
+			clearPassField();
```
Comment (answer261) [inline script/script.js:54]: redundant function execution

File: script/script.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+			return true;
+		}
+		else{
+			alert('Password do not match');
```
Comment (answer261) [inline script/script.js:58]: Should style input border in red, not an alert

File: script/script.js (hunk 3 above)
Comment (AndriySikora) [inline script/script.js:54]: this one clearPassField();?

Comment (answer261) [inline script/script.js:54]: yes, because you have one on line 56

File: script/script.js [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+function clearInputName(){
+	document.getElementById('name').value = '';
+}
+
+function checkEmail(email){
+	var emailField = document.getElementById('email').value;
+	var regex = /^[\w\-\.\+]+\@[a-zA-Z0-9\. \-]+\.[a-zA-z0-9]/;
+	var temp = regex.test(emailField);
+	if (!temp){
+		alert('Your email is invalid');
```
Comment (answer261) [inline script/script.js:30]: add a error border everyware :)

**Pull request merged**